test/integration.test.ts

```diff
@@ -156,23 +157,19 @@ describe('Starting cardano-wallet (and its node)', () => {
   );
 
   describe('Child process logging support', () => {
-    const tmpDir = './test/.tmp';
-    const logFile = path.join(tmpDir, 'logs');
+    let logFile: FileResult;
 
     beforeEach(async () => {
-      await ensureDir(tmpDir);
-      await remove(logFile);
-      await writeFile(logFile,'');
+      logFile = await tmp.file()
     });
 
-    afterEach(async () => {
-      await remove(tmpDir)
+    afterEach(() => {
+      logFile.cleanup()
     });
 
     it('Accepts a WriteStream, and pipes the child process stdout and stderr streams', async () => {
-      const childProcessLogWriteStream = createWriteStream(logFile);
-      const statsBefore = await stat(logFile);
-      expect(statsBefore.mtimeMs === statsBefore.birthtimeMs);
+      const childProcessLogWriteStream = createWriteStream(logFile.path);
```

Comment:
I've not tested it but suspect that this will fail on windows because the temp file is already open for writing.

```suggestion
      const childProcessLogWriteStream = createWriteStream(logFile.path, { fd: logFile.fd });
```